monitoring/views/database_sizes_mb.view.lkml

```diff
@@ -0,0 +1,58 @@
+# If necessary, uncomment the line below to include explore_source.
+include: "//looker-hub/fenix/explores/metrics.explore.lkml"
+
+view: database_sizes_mb {
+  derived_table: {
+    explore_source: metrics {
+      column: metrics__memory_distribution__glean_database_size__sum {}
+      column: clients {}
+      derived_column: database_size_sum_mb {
+        sql: (metrics__memory_distribution__glean_database_size__sum)/1024/1024 ;;
+        }
+      derived_column: database_size_tiers{
+        sql: (metrics__memory_distribution__glean_database_size__sum)/1024/1024;;
+        }
+
+      filters: {
+        field: metrics.channel
+        value: "mozdata.fenix.metrics"
+      }
+      filters: {
+        field: metrics.submission_date
+        value: "28 days"
+      }
+      filters: {
+        field: metrics.metrics__memory_distribution__glean_database_size__sum
+        value: "NOT NULL"
+      }
```

Comment:
generally for filters it's best practice to define filters on an explore level. Having them inside of the view definition hides some information from the user (e.g. why does the data only show me the last 28 days?)

Reply:
I was thinking about this and I'll make the updates